openreview/venue_request/venue_request.py

```diff
@@ -1134,11 +1134,21 @@ def setup_request_form(self):
                 'required': True,
                 'order': 6
             },
+            'publication_chairs': {
+                'description': 'Will your venue have Publication Chairs? The Publication Chairs will only have access to accepted submissions (including author names) and the author accepted group in order to email authors of accepted submissions.',
+                'value-radio': [
+                    'Yes, our venue has Publication Chairs',
+                    'No, our venue does not have Publication Chairs'
+                ],
+                'required': True,
+                'default': 'No, our venue does not have Publication Chairs',
```

Comment:
I think we shouldn't use a default value here so we force the PCs to reply this question. They can miss it and request for publication chairs later.

Reply:
Yes, you are right! This is epecially helpful when the field is added when the PCs use the Revision button, so we force them to pick an option. Changed!